src/Target.cpp

```diff
@@ -165,19 +232,23 @@ Target calculate_host_target() {
         }
         if ((info2[1] & avx512) == avx512) {
             initial_features.push_back(Target::AVX512);
+            // FIXME: port to family/model -based detection.
```

Comment:
Nit: there are existing `FIXME` in Halide, but generally we prefer `TODO`, I think